### DISCUSSION TITLE
scale / tempo division per track

### DISCUSSION THREAD

**Discussion by yoyz** [bangcorrupt/hacktribe#127] [Ideas]
Not an issue, but a whishlist !

I know it might be difficult to implement, but I really enjoy the scale per track on elektron box.
And I would have enjoy it if Korg made it the right way and implement it.

If this e2 could have a "scale per track" setting to divide by 2, 4  the tempo of a given track ( keeping the other track at the same speed by the way ) it could raise the lenght of a given pattern by an order of magnitude and make it really more enjoyable to use.

If you feel to do that, that would be awesome !

**Comment by bangcorrupt** [comment]
I like this idea.  When I read the title I thought you meant different pitch scales per part.  Both would be cool.

**Reply by ecco00** [reply]
do you think it would be easier or a whole different ballpark increasing the bar lenght to 128 steps?

**Reply by bangcorrupt** [reply]
See [this thread](https://github.com/bangcorrupt/hacktribe/discussions/81#discussioncomment-1982035).

**Reply by ecco00** [reply]
ok i'll post the question about how feasible it is in comparison with the scale/tempo division per track there
